### PR TITLE
Callum

### PR DESCRIPTION
**Pull request by caggggggggggggg**

### CODE REVIEW

File: docs/blocks/defense/copper-wall.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Useful for protecting the core and turrets in the first few waves."*
...
+|Internal Name|`callum-wall`|
 |Solid|Yes|
-|Health|320    |
-|Size|1x1  |
-|Build Time|0.25  seconds  |
-|Build Cost|<a href="/wiki/items/copper"><img id="spr" src="/wiki/images/item-copper-xlarge.png"/></a>x6  |
+|Health|10000000    |
+|Size|2x2 |
+|Build Time|0.01  seconds  |
+|Build Cost|<a href="/wiki/items/copper"><img id="spr" src="/wiki/images/item-copper-xlarge.png"/></a>x0  |
```
Comment (KK6DOC) [inline docs/blocks/defense/copper-wall.md]: WTF

**Comment by Anuken** [comment]
no

**Comment by DeltaNedas** [comment]
egg

**Comment by fuzzbuck** [comment]
c a l l u m    w a l l

**Comment by KK6DOC** [comment]
lmao

**Comment by mRebornM** [comment]
what KK6DOC said

**Comment by kcinnu** [comment]
wat even

**Comment by kapzduke** [comment]
calcium

**Comment by KK6DOC** [comment]
It's a called a idiot

**Comment by DeltaNedas** [comment]
https://github.com/DeltaNedas/strokedustry

**Pull request closed without merging**